docs/ref-manual/cli/options.md

```diff
@@ -61,21 +61,28 @@ Adding a message makes it easier to track several runs. It is very useful if you
 To create the message above, we used  
 `certoraRun Bank.sol --verify Bank:Bank.spec --msg 'Removed an assertion'`
 
-### `--rule`
+### `--rule (alternatively: --rules)`
```

Comment:
I think it would be better to actually include two separate headers (### `--rule` followed by ### `--rules`) so that other places that refer to the section can just specify one or the other.  It's a little strange in context, but more convenient elsewhere

Reply:
done